src/main/java/org/breedinginsight/brapi/v2/BrAPIV2Controller.java

```diff
@@ -186,6 +187,15 @@ public HttpResponse<?> getCatchall(@PathVariable("path") String path, @PathVaria
         return executeRequest(path, programId, request, "GET");
     }
 
+    @Delete("/${micronaut.bi.api.version}/programs/{programId}" + BrapiVersion.BRAPI_V2 + "/{+path}")
```

Comment:
When the delete pass through is overridden by specific implementations in future cards, is the plan to use the DeltaBreed external reference id rather than the DbId of the brapi service?